CorosHeader header
int32 message_code
int32 agent_id
string agent_address
int32 agent_port
float64 source_x
float64 source_y
float64 target_x
float64 target_y

